src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -156,82 +156,81 @@ <h2 class="text-muted">
-        </tr>
-      </thead>
-      <tbody>
-        <tr *ngFor="let course of softDeletedCourses">
-          <td>{{course.id}}</td>
-          <td>{{course.name}}</td>
-          <td container="body" [ngbTooltip]="course.createdAt | date:'EEE, dd MMM yyyy, h:mma'">
-            {{course.createdAt | date:'d MMM yyyy'}}
-          </td>
-          <td container="body" [ngbTooltip]="course.deletedAt | date:'EEE, dd MMM yyyy, h:mma'">
-            {{course.deletedAt | date:'d MMM yyyy'}}
-          </td>
-          <td class="text-center">
-            <button class="btn btn-light btn-sm" *ngIf="course.canModifyCourse"
-                (click)="onRestore(course.id)"
-                ngbTooltip="Restore the deleted course and its corresponding students and sessions">
-              Restore
-            </button>
-            <button class="btn btn-light btn-sm disabled" *ngIf="!course.canModifyCourse">
-              Restore
-            </button>
-            <button class="btn btn-light btn-sm" *ngIf="course.canModifyCourse" style="color: red"
-                (click)="onDeletePermanently(course.id)"
-                ngbTooltip="Permanently delete the course and its corresponding students and sessions">
-              Delete Permanently
+<div class="row course-section margin-top-30px" *ngIf="softDeletedCourses.length">
+  <div class="col-12">
+   <h2 class="text-muted">
+      <span class="fa fa-trash-alt"></span> Deleted Courses
+    </h2>
+    <div class="card bg-light top-padded">
+      <div class="card-header bg-secondary recycle-bin-header" (click)="isRecycleBinExpanded = !isRecycleBinExpanded">
+        <div class="row align-items-center">
+          <div class="col-6 text-white">
+            <b>Recycle Bin</b>
+          </div>
+          <div class="col-6 text-right">
+            <button class="btn btn-light btn-sm no-margin" *ngIf="canRestoreAll" (click)="onRestoreAll()"
+                    ngbTooltip="Restore all deleted courses and their corresponding students and sessions">
+              <i class="fas fa-check"></i> Restore All
             </button>
-            <button class="btn btn-light btn-sm disabled" *ngIf="!course.canModifyCourse" style="color: red">
-              Delete Permanently
+            <button class="btn btn-light btn-sm disabled no-margin" *ngIf="!canRestoreAll">
+              <i class="fas fa-check"></i> Restore All
+            </button>&nbsp;
+            <button class="btn btn-light btn-sm text-danger no-margin" *ngIf="canDeleteAll" (click)="onDeleteAll()"
+                    ngbTooltip="Permanently delete all courses and their corresponding students and sessions">
+              <i class="fas fa-times"></i> Delete All
             </button>
-          </td>
-        </tr>
-      </tbody>
-    </table>
+            <button class="btn btn-light btn-sm disabled text-danger no-margin" *ngIf="!canDeleteAll">
+              <i class="fas fa-times"></i> Delete All
+            </button>&nbsp;
```

Comment:
Instead of manually adding `&nbsp;` to add spacing between the buttons, let's use the `scss` file. You can refer to the feedback session's recycle bin too

Reply:
I don't know how exactly was done in "feedback session's recycle bin" or how it uses the scss file to get this done but I did check a little difference in the whitespaced used there. It was different than &nbsp; or simply adding a space using space bar. I don't how he got that "effect" so I copied the character and replaced &nbsp;. Another thing is that apparently that is actually a "&nbsp" therefore there was not change to commit. I simple merge the new changes from master and pushed them.
![Delete button](https://user-images.githubusercontent.com/20428290/55595805-86425500-5713-11e9-8608-0db3e2b90277.PNG)

Reply:
Ah sorry I saw the wrong button, this is what I was referring to:

https://github.com/TEAMMATES/teammates/blob/9994e1ddeb2944a2073f26ae34edddf7d5fd60d8/src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.scss#L5-L7

The spacing should be done by adding a margin instead of `&nbsp;` (Refer [here](https://forums.asp.net/t/1935274.aspx?Is+it+a+bad+habit+to+use+nbsp+to+generate+space+) to see why). Using margin is good practice because we want to ensure that `html` files are there to show the information, while the `scss` files will handle the styling.

So, you can add remove the `&nbsp;` and `no-margin` class, and change it with your own custom css class and add `margin-right: 0.6em;` in it :)

Do try this out and clarify if anything is unclear!

Reply:
Thank you! think I got it.